mmv1/products/databasemigrationservice/privateconnection.yaml

```diff
@@ -0,0 +1,133 @@
+self_link: 'projects/{{project}}/locations/{{location}}/privateConnections/{{private_connection_id}}'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/database-migration/docs/oracle-to-postgresql/create-private-connectivity-configuration'
+  api: 'https://cloud.google.com/database-migration/docs/reference/rest/v1/projects.locations.privateConnections'
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 60
+      delete_minutes: 60
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+autogen_async: true
+description: |
+  The PrivateConnection resource is used to establish private connectivity between Database Migration Service and a customer's network.
+immutable: true
+id_format: projects/{{project}}/locations/{{location}}/privateConnections/{{private_connection_id}}
+import_format:
+  [
+    'projects/{{project}}/locations/{{location}}/privateConnections/{{private_connection_id}}',
+  ]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'database_migration_service_private_connection'
+    primary_resource_id: 'default'
+    vars:
+      private_connection_id: 'my-connection'
+      network_name: 'my-network'
+    test_vars_overrides:
+      network_name: 'acctest.BootstrapSharedTestNetwork(t, "dbms-privateconnection")'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: templates/terraform/constants/dbms_private_connection.go.erb
+  post_create: templates/terraform/post_create/private_connection.go.erb
+  post_import: templates/terraform/post_import/private_connection.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: privateConnectionId
+    description: |-
+      The private connectivity identifier.
+    required: true
+    immutable: true
+    url_param_only: true
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    description: |
+      The name of the location this private connection is located in.
+    required: true
+    immutable: true
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
+    description: The resource's name.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: Labels.
+  - !ruby/object:Api::Type::String
+    name: 'displayName'
+    required: true
+    description: Display name.
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      State of the PrivateConnection.
+    output: true
+    values:
```

Comment:
this is unused for output-only fields. You can use a String-type field with no values instead.